csv/csv_stringify_stream.ts

```diff
@@ -74,33 +74,22 @@ export class CsvStringifyStream<TOptions extends CsvStringifyStreamOptions>
    * @param options Options for the stream.
    */
   constructor(options?: TOptions) {
-    const {
-      separator,
-      columns = [],
-    } = options ?? {};
+    const { separator, columns = [] } = options ?? {};
 
     super(
       {
         start(controller) {
           if (columns && columns.length > 0) {
-            try {
-              controller.enqueue(
-                stringify([columns], { separator, headers: false }),
-              );
-            } catch (error) {
-              controller.error(error);
-            }
-          }
-        },
-        transform(chunk, controller) {
-          try {
             controller.enqueue(
-              stringify([chunk], { separator, headers: false, columns }),
+              stringify([columns], { separator, headers: false }),
             );
-          } catch (error) {
-            controller.error(error);
           }
         },
+        transform(chunk, controller) {
+          controller.enqueue(
+            stringify([chunk], { separator, headers: false, columns }),
```

Comment:
IIUC, throwing naturally achieves the same result as the previous try/catch block.

Reply:
👍 

ref: https://github.com/web-platform-tests/wpt/blob/e67bfa03fc6a7028cc04178d30e67d46132ef0e0/streams/transform-streams/errors.any.js